src/FFI.php

```diff
@@ -315,6 +315,9 @@ private static function init(): void
             }
             $msg .= ". Make sure that you've installed libvips and that '$vips_libname'";
             $msg .= " is on your system's library search path.";
+            if ('preload' === ini_get('ffi.enable')) {
```

Comment:
How about moving this up a bit? On line 284 we have:

```php
        if (!ini_get('ffi.enable')) {
            throw new Exception("ffi.enable not set to 'true'");
        }
```

We could change that to:

```php
        if (!ini_get('ffi.enable') ||
            ini_get('ffi.enable') === 'preload') {
            throw new Exception("ffi.enable not set to 'true'");
        }
```

I think (I think??) we need both tests.